packages/manager/modules/pci/src/projects/project/storages/databases/components/tags-input/tags-input.controller.js

```diff
@@ -0,0 +1,36 @@
+import remove from 'lodash/remove';
+
+export default class {
+  $onInit() {
+    this.items = [];
+    this.tag = '';
+    this.model.forEach((element) => {
+      this.items.push({ title: element });
+    });
+  }
+
+  addTag() {
+    if (!this.isDisabled()) {
+      this.model.push(this.tag);
+      this.items.push({ title: this.tag });
+      this.tag = '';
+    }
+  }
+
+  removeTag(tag) {
+    this.model.pop(tag);
```

Comment:
pop doesn't takes any parameter, do you want to remove by tag or pop last item here ?

Reply:
Fixed